### PR TITLE
[ci skip] [skip ci] [cf admin skip] ***NO_CI*** Fix keys

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

The file had duplicate keys, which is fixed in this PR.

### CODE REVIEW

File: conda-forge.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 conda_forge_output_validation: true
-build_platform: {osx_arm64: osx_64}
-conda_forge_output_validation: true
-provider: {linux_aarch64: default, linux_ppc64le: default, win: azure}
+build_platform:
+  osx_arm64: osx_64
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
```
Comment (bsipocz) [inline conda-forge.yml:6]: what happened to `win`?

```suggestion
  linux_ppc64le: default
  win: azure
```

Comment (BastianZim) [reply conda-forge.yml:6]: Windows now defaults to azure so this key shouldn’t be set anymore. 
It’s just a leftover thing.

Comment (BastianZim) [reply conda-forge.yml:6]: Windows now defaults to azure so this key shouldn’t be set anymore. 
It’s just a leftover thing.

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
Thanks!

**Pull request merged**